server/apolloassistant/verification.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+from Crypto.PublicKey import RSA
+from Crypto.Signature import PKCS1_PSS
+from Crypto.Hash import SHA
+
+
+def generate_RSA_pair(size=2048):
+    private_key=RSA.generate(size)
+    rsa_pair={
+        "private_key" : private_key,
+        "public_key" : private_key.publickey()
+    }
+    for k,v in rsa_pair.items():
+        with open(f"{os.getcwd()}/.envs/{k}.pem","w") as pem_file:
+            pem_file.write(v.exportKey("pem"))
+
+
+def sign(message):
+    h = SHA.new()
+    h.update(message)
+    with open(f"{os.getcwd()}/.envs/private_key.pem", "r") as pem_file:
```

Comment:
Please create `settings.RSA_KEYS_PATH` and setup it relative to `APPS_DIR` and in here use `from django.conf import settings`.